### PR TITLE
Update instruction link & set cookie

### PR DESCRIPTION
**Pull request by arnas**

### CODE REVIEW

**Review by rokeras — DISMISSED** [review]

File: omnisend/includes/Internal/class-sync.php [hunk 1]
```diff
@@ -20,31 +20,13 @@ class Sync {
...
-	 *
-	 * @param $user_id
-	 * @return void
-	 */
-	public static function hook_profile_update( $user_id ): void {
-		if ( \Omnisend_Core_Bootstrap::is_omnisend_woocommerce_plugin_connected() ) {
-			return; // do not sync if omni woo plugin is active.
-		}
-
+	public static function hook_identify_user_by_id( $user_id ): void {
```
Comment (nerijuszaniauskas) [inline omnisend/includes/Internal/class-sync.php:23]: Function prefix `hook_` has special meaning. It means that this function is registered and listens for WP hook without prefix. Please read function description.

Now `hook_identify_user_by_id` description is not correct. Please change to what function actually does like:
`sync_and_identify`

File: omnisend/includes/Internal/class-snippet.php [hunk 2]
```diff
@@ -11,10 +12,17 @@
...
 
+
 	public static function add() {
 		$brand_id = Options::get_brand_id();
 		if ( $brand_id ) {
 			require_once __DIR__ . '/../../view/snippet.html';
 		}
 	}
+
+	public static function set_contact_cookie_id( $contact_id ) {
```
Comment (nerijuszaniauskas) [inline omnisend/includes/Internal/class-snippet.php:23]: Maybe better place for this function in Sync package as it has nothing to do with snipper. If did not know will not look for function `set_contact_cookie_id` in Snippet class.

File: omnisend/class-omnisend-core-bootstrap.php [hunk 3]
```diff
@@ -55,11 +57,20 @@ public static function load(): void {
...
-		add_action( OMNISEND_CORE_CRON_SYNC_CONTACT, 'Omnisend\Internal\Sync::sync_contacts' );
-		add_action( 'user_register', 'Omnisend\Internal\Sync::hook_user_register' );
-		add_action( 'profile_update', 'Omnisend\Internal\Sync::hook_profile_update' );
+			add_action( 'user_register', 'Omnisend\Internal\Sync::hook_identify_user_by_id' );
+			add_action(
+				'wp_login',
+				function ( $user_login, $user ) {
+					Omnisend\Internal\Sync::hook_identify_user_by_id( $user->ID );
+				},
+				10,
```
Comment (nerijuszaniauskas) [inline omnisend/class-omnisend-core-bootstrap.php:67]: Please remove those additional params. 10 is default, 2 is not correct.

Comment (arnas) [reply omnisend/class-omnisend-core-bootstrap.php:67]: 2 should be correct https://developer.wordpress.org/reference/hooks/wp_login/

File: omnisend/includes/Internal/class-snippet.php (hunk 2 above)
Comment (arnas) [reply omnisend/includes/Internal/class-snippet.php:23]: Hmm, I feel that it should be in snippet package as we are setting cookie for snippet

File: omnisend/class-omnisend-core-bootstrap.php (hunk 3 above)
Comment (nerijuszaniauskas) [reply omnisend/class-omnisend-core-bootstrap.php:66]: This is number of argument callback (your function) accepts. `identify_user_by_id` accepts only one argument. So default values are ok, and those 10, 2 can be deleted.

**Review by nerijuszaniauskas — DISMISSED** [review]

File: omnisend/class-omnisend-core-bootstrap.php (hunk 3 above)
Comment (arnas) [reply omnisend/class-omnisend-core-bootstrap.php:66]: My callback function is an anonymous function that accepts two arguments 
`function ( $user_login, $user ) {`. 

I believe I tried creating `add_action` without these arguments and it failed.

Comment (nerijuszaniauskas) [reply omnisend/class-omnisend-core-bootstrap.php:66]: ohh 🤦 you right 👍

File: omnisend/includes/Internal/class-snippet.php [hunk 4]
```diff
@@ -11,10 +11,17 @@
 
 class Snippet {
 
+
```
Comment (rokeras) [inline omnisend/includes/Internal/class-snippet.php:14]: why?

**Review by rokeras — APPROVED** [review]

File: omnisend/includes/Internal/class-snippet.php (hunk 4 above)
Comment (arnas) [reply omnisend/includes/Internal/class-snippet.php:14]: maybe lint 🤷

**Pull request merged**